src/data_management/scenarios/PaginationBar.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import styles from "./PaginationBar.css";
+
+class PaginationBar extends Component {
+    state = {
+        inputPage: ""
```

Comment:
Probably not needed anymore.
Do you know where it is used?

Reply:
Ah used it in componentDidMount and componentWillReceiveProps but I do think that it is not needed. I will just remove them all and test it if it is still needed.

Reply:
Test it and it is not needed anymore so I removed it.